SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.csproj

```diff
@@ -65,7 +65,7 @@
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
     <PostBuildEvent>mkdir "$(SolutionDir)..\..\..\Binaries\ThirdParty\Improbable\Programs\DeploymentLauncher\"
-copy "$(SolutionDir)\$(ProjectName)\$(OutDir)" "$(SolutionDir)..\..\..\Binaries\ThirdParty\Improbable\Programs\DeploymentLauncher\" /Y</PostBuildEvent>
+xcopy "$(SolutionDir)\$(ProjectName)\$(OutDir)." "$(SolutionDir)..\..\..\Binaries\ThirdParty\Improbable\Programs\DeploymentLauncher\" /q /y</PostBuildEvent>
```

Comment:
Yup, xcopy fails if the path ends with a `\` (`OutputDir` path ends with a `\`), adding a `.` solves that

Reply:
(I changed it from `copy` to `xcopy` to reduce the output spam when running `Setup.bat`, since `copy` will print a line for each file it copies, which is around 20 for the deployment launcher)